File: Data Structures & Algorithms/longest-consecutive-sequence/submission-6.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> us;
        for(int i : nums) us.insert(i);
        int maxRes = 0;
        for(int i : nums) {
            if(us.count(i-1)) continue;

            int res = 1, currVal = i;
            while(us.count(++currVal)) {
                ++res;
            }
            maxRes = max(maxRes, res);
        }
        return maxRes;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-6.java
class Solution 
{
    public int[] twoSum(int[] nums, int target)
    {
        int[] result = new int[2];
        for(int i = 0; i < nums.length; i++)
        {
                    // System.out.println("i is " + i +" and len "+nums.length);
            for(int j = i+1; j < nums.length; j++)
            {
                // System.out.println("j is " + j +" and len "+nums.length);
                if(nums[i] + nums[j] == target)
                {
                    result[0] = i;
                    result[1] = j;
                }
            }
        }
        return result;
    }
}
/*
5,5,5,5,3,4,5,6
4,5

*/